dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -52,6 +52,7 @@
 import com.liferay.portal.PortalException;
 import com.liferay.portal.SystemException;
 import com.liferay.portal.model.User;
+import org.apache.velocity.tools.view.context.ViewContext;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.velocity.tools.view.context.ViewContext'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)